fmriprep/interfaces/bids.py

```diff
@@ -250,7 +254,10 @@ def _run_interface(self, runtime):
             if isdefined(self.inputs.extra_values):
                 out_file = out_file.format(extra_value=self.inputs.extra_values[i])
             self._results['out_file'].append(out_file)
-            _copy_any(fname, out_file)
+            self._results['compression'].append(_copy_any(fname, out_file))
+
+        if len(self._results['compression']) == 1:
+            self._results['compression'] = self._results['compression'][0]
```

Comment:
I guess this is over here, now. So I'll just re-note that you can drop these two lines.